hugegraph-common/src/main/java/com/baidu/hugegraph/config/HugeConfig.java

```diff
@@ -160,4 +142,51 @@ private Object validateOption(String key, Object value) {
     private void checkRequiredOptions() {
         // TODO: Check required options must be contained in this map
     }
+
+    public void save(File copiedFile) throws ConfigurationException {
+        FileHandler fileHandler = new FileHandler(this);
+        fileHandler.save(copiedFile);
+    }
+
+    private static Configuration loadConfigFile(File configurationFile) {
+        E.checkArgument(configurationFile.exists() &&
+                        configurationFile.isFile() &&
+                        configurationFile.canRead(),
+                        "Please specify a proper config file rather than: %s",
+                        configurationFile.toString());
+
+        try {
+            final String fileName = configurationFile.getName();
+            final String fileExtension =
+                  fileName.substring(fileName.lastIndexOf('.') + 1);
+
+            final Configuration conf;
+            final Configurations configs = new Configurations();
+
+            switch (fileExtension) {
+                case "yml":
+                case "yaml":
+                    final Parameters params = new Parameters();
+                    final FileBasedConfigurationBuilder<FileBasedConfiguration>
+                          builder = new FileBasedConfigurationBuilder(
+                                        YAMLConfiguration.class).
+                                        configure(params.fileBased()
+                                        .setFile(configurationFile));
+
+                    final Configuration copy = new BaseConfiguration();
+                    ConfigurationUtils.copy(builder.getConfiguration(), copy);
```

Comment:
not sure why need to copy

Reply:
There is refer to the official demo, maybe can remove the copy logic